Formula/k/kubefirst.rb

```diff
@@ -6,6 +6,14 @@ class Kubefirst < Formula
   license "MIT"
   head "https://github.com/kubefirst/kubefirst.git", branch: "main"
 
+  # Upstream appears to use GitHub releases to indicate that a version is
+  # released (and some tagged versions don't end up as a release), so it's
+  # necessary to check release versions instead of tags.
```

Comment:
```suggestion
  # released, so it's necessary to check release versions instead of tags.
```

I compared the fifty most recent releases to the Git tags and didn't see any tags that didn't have a corresponding release, so we can omit the parenthetical here.

Reply:
👍